test/test_support_projects/EmbeddedInterop.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>EmbeddedInterop</RootNamespace>
+    <AssemblyName>EmbeddedInterop</AssemblyName>
+  </PropertyGroup>
+    <ItemGroup>
+        <PackageReference Include="Microsoft.Office.Interop.Excel" Version="15.0.4795.1000" GeneratePathProperty="true">
+            <ExcludeAssets>all</ExcludeAssets>
+        </PackageReference>
+        <Reference Include="$(PkgMicrosoft_Office_Interop_Excel)\lib\net20\Microsoft.Office.Interop.Excel.dll">
```

Comment:
so is there anything in this binary except for the office types?

Reply:
no, just office types and a wrapping class